SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,64 @@
+header:
+  schema-version: 1.0.0
+  expiration-date: '2023-08-31T10:10:09.000Z'
+  last-updated: '2021-09-01'
+  last-reviewed: '2022-09-01'
+  commit-hash: 4dbf78ebc006ee5f668c0a74876ef8d6db9485be
+  project-url: 'https://github.com/kubernetes-sigs/kueue'
+  project-release: 0.5.1
```

Comment:
since this will have the current release version, let's add a step at https://github.com/kubernetes-sigs/kueue/blob/main/.github/ISSUE_TEMPLATE/NEW_RELEASE.md to update this file during release process

Reply:
I've added `SECURITY-INSIGHTS.yml` in the update step.